content/pom.xml

```diff
@@ -274,7 +274,7 @@
                                         <replace file="target/jcr_root/libs/core/wcm/components/search/v1/search/.content.xml" token=".core-wcm" value="Core Structure" />
 
                                         <replace file="target/jcr_root/libs/core/wcm/components/accordion/v1/accordion/.content.xml" token=".core-wcm" value="Core Content" />
-                                        <replace file="target/jcr_root/libs/core/wcm/components/button/v1/button/.content.xml" token=".core-wcm" value="Core Content" />
+                                        <replace file="target/jcr_root/libs/core/wcm/components/button/v2/button/.content.xml" token=".core-wcm" value="Core Content" />
```

Comment:
We should not expose this in a public group until it's ready for production

Reply:
Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1420/commits/8276d5f09fd31d01840c8e2160e97aa9653f1b3a